src/addon_folder.ts

```diff
@@ -130,6 +130,16 @@ async function tryFindActualAddonFolder(root: string) {
 }
 
 async function folderContainsAddonEntry(folderPath: string) {
+    let tomlPath = path.join(folderPath, "blender_manifest.toml");
+    if (await pathExists(tomlPath)) {
+        try {
+            readTextFile(tomlPath);
```

Comment:
Could probably just return true here instead of also trying to read the file. Without `await` this probably doesn't help much anyway.

Reply:
Do we not want to see if it exists first? Or does path.join do that for us?

Reply:
Oh wait nvm 🙈 we already checked 😅